CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contribute
+
+## Setting up
+
+1. Clone this git repository on your local machine inside a `wp-content/plugins/` directory
+2. Install [PHPUnit](https://phpunit.de/). We recommend using version 4.5, as that's the most current version that is compatible with all PHP versions we check against, and thus Travis is using that version.
+3. Activate the plugin.
+4. Visit `http://local.sitename/glotpress` to start using GlotPress. `local.sitename` is the domain where your local development WordPress site runs, dependent on configuration.
+
+## Submitting patches
+
+Whether you want to fix a bug or implement a new feature, the process is pretty much the same:
+
+0. [Search existing issues](https://github.com/GlotPress/GlotPress-WP/issues); if you can't find anything related to what you want to work on, [open a new issue](#helpful-tips-for-writing-issues).
+1. [Fork](https://github.com/GlotPress/GlotPress-WP/fork) the repository.
+2. Create a branch for each issue you'd like to address. Base your new branch on top of `develop` branch. Branches are named as such: `ISSUEID-keywords` for branches where an issue exists, or `keywords` for a branch without an issue yet. For example issue 216 for adding contributing.md would be `216-add-contributing-md`. The same branch would be named `add-contributing-md` if there wasn't an issue for it. Commit your changes.
```

Comment:
Should probably just remove: 

"The same branch would be named `add-contributing-md` if there wasn't an issue for it."

They should have opened a ticket for it back on step 0 :)